.circleci/unittest/linux_olddeps/scripts_gym_0_13/install.sh

```diff
@@ -37,9 +37,9 @@ git submodule sync && git submodule update --init --recursive
 
 printf "Installing PyTorch with %s\n" "${CU_VERSION}"
 if [ "${CU_VERSION:-}" == cpu ] ; then
-    conda install pytorch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 cpuonly -c pytorch
+    conda install pytorch==2.0.1 torchvision torchaudio cpuonly -c pytorch
```

Comment:
Upgraded to using pytorch 2.0.1 since some functions depended on in this job/test used `torch.vmap()`, which is not available in pytorch 1.13.1

Reply:
no we need 1.13 because that's what polymetis is built on :/ 
https://github.com/facebookresearch/fairo/blob/main/polymetis/polymetis/environment.yml

Reply:
We must be compatible with an env where polymetis is installed to have torchrl work on hardware